orchestrator/orchestrator.go

```diff
@@ -381,8 +412,33 @@ func (s *Orchestrator) startExecutionStream(ctx context.Context) error {
 			}
 		}
 		if err := s.rpc.Unsubscribe(context.Background(), subscriber, query); err != nil {
-			s.ErrC <- err
+			s.logger.Error(err.Error())
 		}
 	}()
 	return nil
 }
+
+func keyvals(proc *process.Process, node *process.Process_Node, parentExec *execution.Execution, event *event.Event, data *types.Struct) []interface{} {
```

Comment:
I would get rid of this function and just and just add the classic data to the logger even if we need to pass the logger in the parameter. I find this function confusing and it wouldn't be a lot of duplication to get rid of this function.
Not really important but might be a lot easier to read the code without it

Reply:
I already tried and the problem is most of the functions are compatible with nil parameter.
It means the `if != nil` tests that this function is doing will have to be duplicated as well.

Reply:
right didn't see that part so yes might be a lot of code not really useful